pkg/querier/querier.go

```diff
@@ -40,6 +43,19 @@ type AgentNetworkPolicyInfoQuerier interface {
 	GetRuleByFlowID(ruleFlowID uint32) *types.PolicyRule
 }
 
+type AgentMulticastInfoQuerier interface {
+	// GetIGMPReportNPStats gets statistics generated by networkpolicies that block or allow
+	// IGMP reports message. The statistics returned are incremental and will be reset after
+	// reporting to antrea-controller.
```

Comment:
"will be reset after each call".

As the stats is reset by the func call, not "after reporting to antrea-controller".

Reply:
updated